### ISSUE TITLE
Import under condition

### ISSUE THREAD

**Issue by balsagoth** [microsoft/pyright#120]
I think this should be ok.

```python
condition = False
if condition:
    import socket
    socket.gethostname()
````

![Screenshot 2019-05-09 18 05 04](https://user-images.githubusercontent.com/183991/57472730-1aa95580-7286-11e9-87eb-908c9049cffa.png)

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next version.

**Comment by erictraut** [comment]
This is now fixed in 1.0.26
